src/components/Content.tsx

```diff
@@ -1,14 +1,35 @@
-import {
-  Box,
-  Breadcrumb,
-  BreadcrumbItem,
-  BreadcrumbLink,
-  Icon,
-} from "@chakra-ui/react";
-import React from "react";
-import { FiChevronRight } from "react-icons/fi";
+import React from 'react';
+import { FiChevronRight } from 'react-icons/fi';
+import { useLocation } from 'react-router-dom';
+
+import { Box, Breadcrumb, BreadcrumbItem, BreadcrumbLink, Icon } from '@chakra-ui/react';
+
+interface Props {
+  children: JSX.Element;
+}
+
+export const Content = ({ children }: Props): JSX.Element => {
+  const location = useLocation().pathname.split("/");
```

Comment:
Faire deux variables séparées pour pouvoir utiliser la location de React Router à un autre endroit
```suggestion
  const location = useLocation();
  const splittedLocation = location.pathname.split("/");
```